examples/Backtesting/botorch_analytical.py

```diff
@@ -36,8 +35,8 @@
 
 SMOKE_TEST = "SMOKE_TEST" in os.environ
 
-N_MC_ITERATIONS = 2 if SMOKE_TEST else 30
-N_DOE_ITERATIONS = 2 if SMOKE_TEST else 15
+N_MC_ITERATIONS = 2 if SMOKE_TEST else 2
```

Comment:
this destroys the purpose of SMOKE_TEST

Reply:
Whuups that was an error that carried over from testing. Will change, thanks for noticing!

Reply:
Fixed.